Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py

```diff
@@ -249,17 +284,18 @@ def aws_recreate_sg(args: Dict[str, Any]) -> str:
     port = int(args.get('port', None))
     protocol = args.get('protocol', None)
     public_ip = args.get('public_ip', None)
+    assume_role = args.get('assume_role', None)
```

Comment:
FYI, i readded these because it was causing MyPy errors and don't seem to hurt anything
![2023-09-06_09-24-27](https://github.com/demisto/content/assets/32227961/6a390ae9-8059-44e4-9d15-bd6ea0f17fcb)